pabot/pabot.py

```diff
@@ -283,9 +288,19 @@ def _writer():
         message = MESSAGE_QUEUE.get()
         print message
 
-def _write(message):
+def _write(message, color=None):
+    if color == Color.RED:
+        message = _wrap_with_red(message)
+    if color == Color.GREEN:
+        message = _wrap_with_green(message)
     MESSAGE_QUEUE.put(message)
 
```

Comment:
I think something like this would be better:

    if color:
        message = _wrap_with(color, message)

    def _wrap_with(color, message):
        return "%s%s%s" % (color, text, Color.ENDC)